transifex/requirements/edx-platform.txt

```diff
@@ -0,0 +1,11 @@
+Babel==1.3
+Django==1.10.7
+django-babel-underscore==0.5.2
+edx-i18n-tools==0.4.2
+mako==1.0.7
+PyGithub==1.29
+PyYAML==3.12
+transifex-client==0.12.4
+
+# Python libraries to be installed directly from github
+git+https://github.com/edx/django-wiki.git@b232cff63c5d342a5fe8f11a25f03803c7c45d65#egg=django-wiki
```

Comment:
Can you clarify what this is needed for?

Reply:
Yes this includes a code refactor.

Reply:
Yes this was to fix a import error.